en/docs/services/create-your-first-service.md

```diff
@@ -214,6 +214,47 @@ You can view statistics related to the service as the following diagram illustra
 The low-code diagram displays the success rate and the latency for each connector. 
 The Observability view to the right of the low-code diagram displays graphs that depict the throughput and the latency. You can move 
 the pointer over the throughput graph to view details relevant for specific times. If you want to drill down further, 
-click the **Diagnostics View** tab.  
+click the **Diagnostics View** tab.
 
-Congratulations! You have now successfully created a service, tested it, deployed it, and observed its statistics.
+The requests you sent to generate the above statistics use the temporary API key created when you deploy the service. To access the API of the `CovidStatus` service, users of the service need to generate their credentials. To allow this, proceed to the next step and publish the API.
+
+## Step 9: Publish the service API
+
+To publish the API of the `CovidStatus` service, follow this procedure:
+
+1. Open the left navigator and click **APIs**.
```

Comment:
Should we say "On the left navigator" instead of "open the left navigator"? WDYT?

Reply:
At this point, the navigator is not visible by default. So the user needs to click on the navigator icon and open it.

Reply:
Ack